ros2_batch_job/__main__.py

```diff
@@ -402,6 +438,11 @@ def run(args, build_function, blacklisted_package_names=None):
         job.run(['"%s"' % job.python, '-m', 'pip', '--version'], shell=True)
         # Install pip dependencies
         job.run(['"%s"' % job.python, '-m', 'pip', 'install', '-U'] + pip_dependencies, shell=True)
+        if sys.platform != 'win32':
+            colcon_script = os.path.join(venv_path, 'bin', 'colcon')
+        else:
+            colcon_script = 'c:\\python36\\Scripts\\colcon.exe'
```

Comment:
Does this mean that we will now need to update the ci scripts on new python minor version?
Could we infer that from the environment ?

Reply:
If that can be done automatically even better.

Reply:
Fixed in 4fe9cfdefb4ae828ce8599a6d4ef8318be5c27cf.